packages/app/universal-ts-utils/src/array/uniq.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { describe, expect, it } from 'vitest'
+import { uniq } from './uniq'
+
+describe('uniq', () => {
+	it('returns a new array of mixed primitive value without duplicates', () => {
```

Comment:
would be nice to test with: nulls, undefineds, NaN, objects

Reply:
Good point. I added a few more values into the inputs. Let me know if there are some others you would test for.